tests/RobotFramework/tests/customizing/tedge_init.robot

```diff
@@ -0,0 +1,104 @@
+    ...    this test step is confirming the deletion of the folders
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/mosquitto-conf
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/operations
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/plugins
+    ThinEdgeIO.Directory Should Not Exist    /etc/tedge/device-certs
+    ThinEdgeIO.Directory Should Not Exist    /var/tedge
+    ThinEdgeIO.Directory Should Not Exist    /var/log/tedge
+
+Tedge init and check creation of folders
+    [Documentation]    Running tedge init should create these folders
+    ...    this test step is confirming the creation of the folders
+    Execute Command    sudo tedge init
+    ThinEdgeIO.Directory Should Exist    /etc/tedge
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/mosquitto-conf
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/operations
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/plugins
+    ThinEdgeIO.Directory Should Exist    /etc/tedge/device-certs
+    ThinEdgeIO.Directory Should Exist    /var/tedge
+    ThinEdgeIO.Directory Should Exist    /var/log/tedge
+
+Check ownership of the folders
+    [Documentation]    Running tedge init has created the folders assigning default user/group
+    ...    this test step is confirming the default values for user/group
+    ${etc_tedge}   Execute Command    ls -ld /etc/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge}    \s*tedge:root\s*
+    ${etc_tedge_mosq}   Execute Command    ls -ld /etc/tedge/mosquitto-conf | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_mosq}    \s*mosquitto:mosquitto\s*
+    ${etc_tedge_operat}   Execute Command    ls -ld /etc/tedge/operations | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_operat}    \s*tedge:tedge\s*    
+    ${etc_tedge_plug}   Execute Command    ls -ld /etc/tedge/plugins | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_plug}    \s*tedge:tedge\s*    
+    ${etc_tedge_devcert}   Execute Command    ls -ld /etc/tedge/device-certs | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge_devcert}    \s*mosquitto:mosquitto\s*
+    ${var_tedge}   Execute Command    ls -ld /var/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_tedge}    \s*tedge:tedge\s*
+    ${var_log_tedge}   Execute Command    ls -ld /var/log/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${var_log_tedge}    \s*tedge:tedge\s*
+
+Change user/group and check the change
+    [Documentation]    Running tedge init --user <user> --group <group>  is setting custom user/group
+    ...    this test step is confirming the custom values for user/group
+    Execute Command    sudo tedge init --user petertest --group petertest
+    ${etc_tedge}   Execute Command    ls -ld /etc/tedge | awk '{print $3 ":" $4}'
+    Should Match Regexp    ${etc_tedge}    \s*petertest:root\s*
```

Comment:
Can't run the `ls command and matching` be moved to a function and just pass the `directory or file` name to it?. This will reduce a lot of code duplication.